tests/unit/LHNOrderTest.js

```diff
@@ -158,18 +158,37 @@ Onyx.init({
 });
 
 function getDefaultRenderedSidebarLinks() {
+    class ErrorBoundary extends React.Component {
+        // Error boundaries have to implement this method. It's for providing a fallback UI, but
+        // we don't need that for unit testing, so this is basically a no-op.
```

Comment:
**question:** Why do we need the error boundary?

Reply:
Is that a rhetorical question or something you would like more details on in the comment? I only ask because we talked about this face-to-face and I thought you understood about the error boundary being added and Jest somethingsomething swallowing up errors.

Reply:
I've added a big code comment to explain the `why` behind why this is necessary.

Reply:
No, it wasn't rhetorical. I was curious why it is needed (did not recall the previous conversation) and appreciate the code comment so that others with the same question will have a better idea.

**thought:** Feels like this could be moved into some kind of `utils` so we can re-use this in other tests. But NAB, since this is one of the few tests we have that renders a component.